src/main/java/org/redhat/mdcr/route/ProducerRouteBuilder.java

```diff
@@ -7,20 +7,26 @@
 @Component
 public class ProducerRouteBuilder extends RouteBuilder {
 
-	@Value("${queue.name}")
+	@Value("${producer.queue.name}")
 	private String queueName;
 	
+	@Value("${producer.route.switch}")
+	private boolean runRoute;
+	
 	@Override
 	public void configure() throws Exception {
 
 		onException(Exception.class)
-			.handled(true)
-			.log("******#####Exception occured. Body is ${body}");
+				.handled(true)
+				.transform(simple("${exception.message}"))
+				.log("Exception: ${body}");
+
+		if (runRoute) {
```

Comment:
I think we can do this with `.autoStartup(runRoute)`.

https://camel.apache.org/configuring-route-startup-ordering-and-autostartup.html